spec/features/granules/granule_filters_spec.rb

```diff
@@ -108,6 +108,31 @@
         click_button "granule-filters-submit"
       end
 
+      it "entering and removing values doesn't set invalid query param in the url" do
+        page.execute_script("document.getElementById('cloud-cover-min').scrollIntoView()")
+        fill_in "Minimum:", with: "2.5"
+        page.find(".master-overlay-secondary-content").click
+        wait_for_xhr
+        page.save_screenshot "1.png"
+
+        project = Project.find(URI.parse(page.current_url).query[/^projectId=(\d+)$/, 1].to_i)
+        p "--------- #{project.path}"
+
+        page.execute_script("document.getElementById('cloud-cover-min').value = ''; null;")
+        fill_in "Minimum:", with: ""
+        # keypress_script = "var e = $.Event('keypress', { which: 9 }); $('#cloud-cover-min').trigger(e); null;"
+        # page.execute_script(keypress_script)
+        expect(page).to have_field("Minimum", with: "")
+        page.find(".master-overlay-secondary-content").click
+        wait_for_xhr
+        page.save_screenshot "2.png"
+        click_button "Apply"
```

Comment:
The ```Minimum``` input loses focus when the button is clicked. Manually, this will trigger query param to change.